src/main/kotlin/no/nav/familie/ef/søknad/mapper/kontrakt/BarnMapper.kt

```diff
@@ -28,7 +28,9 @@ object BarnMapper {
                              fødselTermindato = barn.fødselsdato?.tilSøknadsDatoFeltEllerNull(),
                              terminbekreftelse = dokumentfelt(TERMINBEKREFTELSE, vedlegg),
                              annenForelder = mapAnnenForelder(barn.forelder),
-                             samvær = mapSamvær(barn.forelder, vedlegg))
+                             samvær = mapSamvær(barn.forelder, vedlegg),
+                             skalHaBarnepass = barn.skalHaBarnepass?.tilSøknadsfelt(),
+                             barnepass = barn.barnepass?.let { Søknadsfelt("", BarnepassMapper.tilBarnepass(it)) }) //TODO
```

Comment:
Label ikke klart fra UI?

Reply:
Vet ikke hvordan dere har gjort med disse tidligere.
Eks "Årsak til alene med barn" som finnes i koden i soknad-api finnes ikke i frontend.

Reply:
Fint om så mye som mulig kommer fra frontend, men dersom det ikke kommer inn kan vi hardkode i backend. Ikke ideelt, særlig med tanke på oversetting senere. I utgangspunktet synest jeg vi skal sjekke med frontend og høre om det er problematisk å legge på label.

Reply:
@charliemidtlyng og @miichu - er noen steder i api vi gjerne skule hatt label/verdi - mye jobb å legge på dette?

Reply:
barnepass er en liste - en liste har vel per def ikke label?
Hvordan har vi løst mapping av barn (som også er en liste?)

Reply:
Ser ut som lister har hardkodet label i api-et for overgangsstønad. Vet ikke om det er noen smart og enkel måte å få sendt dette fra frontend?

Reply:
@ma10s  Hvor er det label og verdi mangler? 
Jeg mener at alle feltene i IBarnepass skl ha label og verdi. 🤔

Reply:
ÅrsakBarnepass blir fylt ut i den nye PRen jeg har for validering, men den er ikke helt klar ennå :)